custom_components/opnsense/coordinator.py

```
@@ -119,144 +106,159 @@
+                    .get(server_name, {})
+                )
+
+                for prop_name in [
+                    "total_bytes_recv",
+                    "total_bytes_sent",
+                ]:
+                    try:
+                        current_parent_value: float = server[prop_name]
+                        previous_parent_value: float = previous_server[prop_name]
+                        change: float = abs(
+                            current_parent_value - previous_parent_value
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)